commit 30cdc9e86783f6d7b8946a07c52957dfa307340e
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Feb 17 15:26:13 2017 +0000

    brcmfmac: Mute expected startup 'errors'
    
    The brcmfmac WiFi driver always complains about the '00' country code.
    Modify the driver to ignore '00' silently.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
index bc78593a611b..604744f7af9b 100644
--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/cfg80211.c
@@ -6786,6 +6786,8 @@ static void brcmf_cfg80211_reg_notifier(struct wiphy *wiphy,
 	/* ignore non-ISO3166 country codes */
 	for (i = 0; i < sizeof(req->alpha2); i++)
 		if (req->alpha2[i] < 'A' || req->alpha2[i] > 'Z') {
+			if (req->alpha2[0] == '0' && req->alpha2[1] == '0')
+				return;
 			brcmf_err("not a ISO3166 code (0x%02x 0x%02x)\n",
 				  req->alpha2[0], req->alpha2[1]);
 			return;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1151_30cdc9e86783f6d7b8946a07c52957dfa307340e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1151_30cdc9e86783f6d7b8946a07c52957dfa307340e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
